search-backend-remoterepository/src/test/java/org/apache/maven/search/backend/remoterepository/internal/RemoteRepositorySearchBackendImplTest.java

```diff
@@ -172,6 +173,21 @@ public void gav() throws IOException {
         dumpPage(searchResponse);
     }
 
+    @Test
+    public void gave() throws IOException {
+        // LIST GAVCEs
+        SearchRequest searchRequest = new SearchRequest(BooleanQuery.and(
+                FieldQuery.fieldQuery(MAVEN.GROUP_ID, "org.apache.maven.plugins"),
+                FieldQuery.fieldQuery(MAVEN.ARTIFACT_ID, "maven-clean-plugin"),
+                FieldQuery.fieldQuery(MAVEN.VERSION, "3.1.0"),
+                FieldQuery.fieldQuery(MAVEN.FILE_EXTENSION, "jar")));
+        RemoteRepositorySearchResponse searchResponse = backend.search(searchRequest);
+        assertThat(searchResponse.getTotalHits(), equalTo(1));
+        assertNotNull(searchResponse.getPage().get(0).getLastUpdated());
```

Comment:
I would to like a use one framework ...
so hamcrest or Assertions from JUnit in all place in one test class